swap_meet/vendor.py

```diff
@@ -1,2 +1,74 @@
+from .item import Item
+
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        else:
+            return False
```

Comment:
We can get rid of the `else` conditional by using a guard clause. This allows the main focus of the function body (removing of the item) to be less indented.

```python
    def remove(self, item):
        if item not in self.inventory:
            return False
        self.inventory.remove(item)
        return item
```